python/hsfs/feature_view.py

```diff
@@ -319,6 +439,36 @@ def create_training_data(
     ):
         """Create a training dataset and save data into `location`.
 
+        !!! example "Create random splits"
+            ```python
+            feature_view = fs.get_feature_view(name="feature_view_name")
+
+            version, job = feature_view.create_training_data(
+                description='transactions_dataset_jan_feb',
+                data_format='csv',
+                write_options={"wait_for_job": False}
```

Comment:
add comments also so its easier for the user to follow. Explain and show what the difference between `{"wait_for_job": False}` and `{"wait_for_job": True}`